README.md

```diff
@@ -1 +1,37 @@
 
+For this example ill be using Management Services, which is a logical group of services that are deployed on management hubs (Previously known as Management Hub Components).
+
+2 Repos will coexist:
+- Preprod Management Service Set
+- Prod Management Service Set
+
+ServiceSet - A repo template for Argo ApplicationSet which deploys a set of services defined in values.yaml to a specific cluster.
```

Comment:
What is ApplicationSet? Give some insight to the reader
Do not assume they know everything, at the very least place a link to argo docs for explanation

Reply:
Just added a brief explanation and a link to read more about ApplicationSet